tsconfig.json

```diff
@@ -4,7 +4,8 @@
         "target": "es6",
         "outDir": "out",
         "lib": [
-            "es6"
+            "es6",
+            "dom"
```

Comment:
This isn't really okay as it will break environments where there isn't a DOM (such as Node).